ivy_models_tests/roberta/test_roberta.py

```diff
@@ -0,0 +1,26 @@
+import os
+import ivy
+import pytest
+import numpy as np
+from ivy_models import roberta_base
+
```

Comment:
could you please quickly ref `test_alexnet` and make the model object init happen once only for saving ci resources, thanks!